src/Console/SecurityCommand.php

```diff
@@ -2,9 +2,12 @@
 
 namespace Jorijn\LaravelSecurityChecker\Console;
 
+use Enlightn\SecurityChecker\AdvisoryAnalyzer;
```

Comment:
In light of DRY: There are now three instances where all four dependencies are used. I'd like for this to move into a separate security service that outputs `$checkResult` like below. I believe this makes it easier to maintain and test since we're maintaining just one class instead of three.

Reply:
That's a good point, I'll make sure to PR this into Enlightn for a better API.